src/utils/ConfigLoader.ts

```diff
@@ -1,11 +1,12 @@
 import * as dotenv from 'dotenv';
 
-const DEFAULT_ENV_FILE = `${__dirname}/../../.env`;
+import { getEnv } from 'app/utils/EnvironmentUtils';
+
+const DEFAULT_ENV_FILE = `.env`;
 
 export const loadEnvironmentVariablesFromFiles = () => {
-    dotenv.config({ path: DEFAULT_ENV_FILE });
-    if (process.env.NODE_ENV) {
-        const envFilePath = `${__dirname}/../../.env.${process.env.NODE_ENV}`;
-        dotenv.config({ path: envFilePath });
-    }
+    const env = getEnv('NODE_ENV');
```

Comment:
I believe we should simplify it so only `.env.[env]` is loaded.

Reply:
Okay, and we allow it to fail loudly if the environment variable isn't present?